tests/test_spelling_ally_ly.py

```diff
@@ -0,0 +1,21 @@
+"""Tests for spelling.ally_ly check."""
+
+from proselint.checks.spelling import er_or as chk
```

Comment:
Looks like this is why CI failed unfortunately - we forgot to change the test imports and I didn't pick up on it in review.
If you change these imports to the checks the tests are for, it should work just fine :)